src/shared/ui/image/utils/checkForSuitable.ts

```diff
@@ -0,0 +1,21 @@
+import { MIN_SIZE_BREAKPOINT } from '../constants';
+
+const checkForSuitable = (imgUrl: string, width: number = MIN_SIZE_BREAKPOINT) => {
```

Comment:
Please, add type of return

Reply:
Thanks! if the general approach is approved, I will fix it